nessie/golden/lakectl_annotate_a_rec.golden

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
nit; I think `lakectl_annotate_a_recursive` is clearer

Reply:
Done. Also renamed `lakectl_annotate_top_rec` to `lakectl_annotate_top_recursive`